### PR TITLE
Complete User Guide

### PR DESCRIPTION
**Pull request by potatowagon**
Please do look through the revised command format for each feature. This version solves the problem of
-confusing commands
Added fixed order to parameters

-overlapping commands
Made all parameters in <> compulsory in the command. Negates the need to assume.

-unintuitive date time format
Valid inputs are now closely similar to natural language. Eg. 24oct2017 2pm to 438pm instead of 24/10/2017 1400 24/10/2017 1638

-complicated adding
replaced event, task, deadline reserved keywords for task creation with add. Logic will differentiate the type of task to create based on the number of essential parameters.
Floating Task: 1 parameter (name)
Deadline: 3 parameters (name, date, time)
Event: 5 parameters (name, start date, start time, end date, end time)
Optional parameters will not be confused with essential parameters since they are tagged with [p-] [loc-] in the input

Alternatively, logic can differetiate thetype of task to create by the total number of parameters in input

Floating Task: 1-2 parameter (name, priority)
Deadline: 3 parameters (name, date, time)
Event: 5-6 parameters (name, start date, start time, end date, end time, location)

-Long inputs
Added parameter keywords to shorten some long commonly used words, such as month, today, tommorow
Changed edit/delete/fin to index identificaion instead of name identification

### CODE REVIEW

**Comment by pyokagan** [comment]
@nightism Please read through and take note of the changes as well.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -21,251 +45,323 @@
...
-### Adding a event-like task : `event`
+To
+	
+    to
+    
+ * This keyword is only applicable for adding an event. `to` can only be entered into an `<END_DATE>` field. `to` would mean the `<END_DATE>` is the same as the `<START_DATE>` for an event. See [example](#to).
 
-Adds an event with specific starting and ending date or time to TaskTracker.
+**<a name="TimeFormat">Time Format</a>**
+The 12 hour clock is used. hh:mm am/pm
```
Comment (pyokagan) [inline docs/UserGuide.md:251]: So here it says we are using `hh:mm`, but below I see `830am` ????

By the way, we need to change all time formats to use `hh:mm` format e.g. `8:30am` due to limitations in Java's `DateTimeFormatter`.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -21,251 +45,323 @@
...
 
-### Adding a floating task : `task`
+<a name="DateFormat">**Date Format**</a>
+Valid input examples:
 
-Adds a floating task to TaskTracker.
+ddmonthyy
 
-    task "FLOATING_TASK_NAME" [PRIORITY]
+	31dec2016
```
Comment (pyokagan) [inline docs/UserGuide.md:219]: I preferred the numbers and slashes. e.g. `31/12/2016`. Can we revert back to that for now?

Comment (potatowagon) [reply docs/UserGuide.md:219]: ok

File: docs/UserGuide.md (hunk 1 above)
Comment (potatowagon) [reply docs/UserGuide.md:251]: ok

**Comment by pyokagan** [comment]
@potatowagon Will there be any more updates to this PR or do you want me to merge this first?

**Comment by potatowagon** [comment]
please merge first. I will fix time format and other issues soon

**Comment by pyokagan** [comment]
@potatowagon OK thanks.

**Comment by pyokagan** [comment]
Merged in 75da627aaa0473ac5c8f23d173d268a11a96647e

There were some formatting errors so I fixed them while merging. Please pull from master and build off it. Thanks.

**Comment by potatowagon** [comment]
okay

File: docs/UserGuide.md (hunk 1 above)
Comment (potatowagon) [reply docs/UserGuide.md:251]: must 8pm be written as 8:00pm for input?

Comment (potatowagon) [reply docs/UserGuide.md:251]: ie. If the minute field is 00, may it be ommited from the command?

Comment (pyokagan) [reply docs/UserGuide.md:251]: Can look at the test cases [here](https://github.com/CS2103AUG2016-T11-C4/main/blob/master/src/test/java/seedu/address/logic/parser/TimeParserTest.java)

In short: can be omitted.

**Pull request closed without merging**